SocketRocket/SRWebSocket.h

```diff
@@ -41,6 +41,12 @@ extern NSString *const SRWebSocketErrorDomain;
 @property (nonatomic, readonly) SRReadyState readyState;
 @property (nonatomic, readonly, retain) NSURL *url;
 
+// Specifies whether SSL trust chain should NOT be evaluated.
+// By default this flag is set to NO, meaning only secure SSL connections are allowed.
+// For DEBUG builds this flag is ignored, and SSL connections are allowed regardless
+// of the certificate trust configuration
+@property (nonatomic, readwrite) BOOL allowUntrustedSSLCertificates;
```

Comment:
Total nit, this should be allowsSelfSignedCertificates (https://developer.apple.com/library/mac/documentation/Cocoa/Conceptual/CodingGuidelines/Articles/NamingIvarsAndTypes.html) -- in the initializers it should be allows just like you've done.

More importantly though, to stop people from hurting themselves, this property should be marked readonly and the only way to specify it is in the initializer.